docs/source/monitors.rst

```diff
@@ -17,24 +17,140 @@ These monitors build off of a previous COS Branch monitoring program, an overvie
 
 Target Acquisition Monitors
 ---------------------------
-The goal of the Target Acquisition monitors is to assist in cases of failed acquisitions as well has keep track of slew
-vs time in order to identify any systematic trends or issues with COS target acquisitions.
-
-Each acquisition monitor is covers both slew directions (ACQSLEWX and ACQSLEWY) and according to which FGS was used.
+The goal of the Target Acquisition monitors is to assist in cases of failed acquisitions as well as keep track of
+performance order to identify any systematic trends or issues with COS target acquisitions.
 
 ACQIMAGE Monitor
 ^^^^^^^^^^^^^^^^
+.. note::
+
+    This monitor is still in development.
+    Output and tracking will be described in more detail once it is finalized.
+
+The ``AcqImage`` monitor is intended to be a more generalized monitor that keeps track of acquisition statuses and slew
+statistics.
 
-.. Needs to be finalized
+This monitor is run at a monthly cadence.
 
 Spectroscopic Acquisition Monitors: ACQPEAKD and ACQPEAKXD
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The Spectroscopic Acquisition Monitors include the ``AcqPeakdMonitor`` and the ``AcqPeakxdMonitor``.
+These monitors are used to track changes in in the precision of the spectroscopic acquisition modes.
+
+Tracking
+++++++++
+Both Spectroscopic Acquisition Monitors track the standard deviation of their respective slew values (ACQSLEWX for
```

Comment:
Replace 'standard deviation' with 'value'

Reply:
Why is that? The `track` method of these two monitors actually do return the standard deviation of the slew values, so they "track" the standard deviation. 

If they should also track other information, let me know though

Reply:
The label on the Y axis says it is the offset, not the standard deviation. I guess I will have to look at this in more detail to understand what is being plotted.

Reply:
Ah, so "tracking" is different from the plotting. The plots have a certain set of information, but the monitors are also meant to look at other results over time. Basically the results from `track` may or may not be on the plots. 

In  this case, it was decided that in addition to the plots that were made, the quantity that we wanted to "track" through time was the standard deviation